src/uwtools/resources/schema_with_metatasks.rng

```diff
@@ -1,10 +1,8 @@
-<!-- Copied from the Rocoto repository at release version 1.3.5.-->
```

Comment:
I cleaned this up with the `tidy` utility (for posterity, with the command `tidy -indent --indent-attributes 1 -quiet --sort-attributes alpha -wrap 0 -xml`) because some of the formatting was wonky, making it difficult to follow the logic as a reference. See e.g. https://github.com/ufs-community/workflow-tools/blob/1cf4220a7f8b261f0c617554666d8fca5900a4cf/src/uwtools/resources/schema_with_metatasks.rng#L413-L435